### DISCUSSION TITLE
Did you test ProxyLite.ru?

### DISCUSSION THREAD

**Discussion by SkullFace141** [engageub/InternetIncome#25] [General]
Did you test ProxyLite.ru?

**Comment by engageub** [comment]
Please enable logs and raise an issue if you see anything on your PC not working. 
Thank you

**Comment by engageub** [comment]
Just an Info below:
When proxylite was run on Arm64 architecture with binfmt, the memory abruptly increased and did not stop. 
This was not the case with other apps used.
So, monitor your memory and cpu when running the apps. 

Thank you

**Comment by SkullFace141** [comment]
how much earning?

**Comment by SkullFace141** [comment]
@engageub

**Comment by engageub** [comment]
The earning is very less when compared to the daily bonus given by the website.
You can earn more from daily bonus than using it on a single connection. However it requires deposit in order to increase the rate of earnings. No deposits are made due to risk factors. You may try to deposit and see if you are getting more income.
Its upto the users choice to use it or not. The script is working as expected and you would see your servers online in the website immediately. 
Earnings have not been estimated yet per day due to daily bonus being more than proxy sharing usage and also the memory was also increasing gradually on arm64. 
You will need to see it yourself since it depends on location and there are not much reviews on this website. 

Thank you

**Comment by SkullFace141** [comment]
I think its scam

**Reply by engageub** [reply]
Could you please confirm if you reached withdrawal limit.

Thank you
